adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -592,6 +596,12 @@ def _perceive_objects(self) -> None:
             self._instantiate_object_schema(
                 only(object_schemata), situation_object=situation_object
             )
+        # Add ground to _object_perceptions here
```

Comment:
Rather than tracking if you saw it on the main loop through, prefer instead to do:
```python
if not any(situation_object.ontology_node == GROUND 
                for situation_object in self._situation.objects):
    ...
```